### ISSUE TITLE
[Bug] vuejs-accessibility/alt-text doesn’t work

### ISSUE THREAD

**Issue by tai-fukaya** [vue-a11y/eslint-plugin-vuejs-accessibility#309]
I set custom image component to img props, like this.
https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility/blob/main/docs/alt-text.md#rule-details
```
{
  "rules": {
    "vuejs-accessibility/alt-text": [
      "error",
      {
        "img": [“CustomImage”],
      }
    ]
  }
}
```
it doesn’t work.

But it work.
```
{
  "rules": {
    "vuejs-accessibility/alt-text": [
      "error",
      {
        "img": [“custom-image”],
      }
    ]
  }
}
```

Is it bug?
Please fix code or update rules page.

**Comment by tai-fukaya** [comment]
@kddnewton 
getElementType return kebeb case, but options[element] does not call makeKebabCase.
it causes that.
https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility/blob/d9ed347064c99664e8299d0be3255dea6863b0ff/src/utils/getElementType.ts#L16
